backend/src/test/java/com/ody/auth/service/AuthServiceTest.java

```diff
@@ -1,51 +1,93 @@
 package com.ody.auth.service;
 
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.ody.auth.token.AccessToken;
+import com.ody.auth.token.RefreshToken;
+import com.ody.common.BaseServiceTest;
+import com.ody.common.TokenFixture;
+import com.ody.common.exception.OdyBadRequestException;
+import com.ody.common.exception.OdyUnauthorizedException;
+import com.ody.member.domain.DeviceToken;
+import com.ody.member.domain.Member;
+import com.ody.member.repository.MemberRepository;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+
+class AuthServiceTest extends BaseServiceTest {
+
+    @Autowired
+    private AuthService authService;
 
-class AuthServiceTest {
+    @Autowired
+    private MemberRepository memberRepository;
 
-    @DisplayName("액세스 토큰을 파싱한다.")
+    @DisplayName("로그 아웃 테스트")
     @Nested
-    class parseAccessToken {
+    class LogOutTest {
```

Comment:
[필수]
`logout`으로 통일해주세요~
```suggestion
    class LogoutTest {
```